src/Core.php

```diff
@@ -39,7 +35,7 @@ public function report(Throwable $e): void
         try {
             $this->sensor->exception($e);
         } catch (Throwable $e) {
-            $this->handleUnrecoverableException($e);
+            Nightwatch::unrecoverableExceptionOccurred($e);
```

Comment:
It is a bit of a circular dependency that the class that powers the facade references the facade.

I've intentionally made the `unrecoverableExceptionOccurred` and `handleUnrecoverableExceptionsUsing` methods static. This means, for these particular functions, there isn't really a circular dependency.

You can interact with them without having the Facade resolve the class that powers it.

Having it on the `Nightwatch` facade was purely an API aesthetics choice.